Add jobs aa1 to master

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently there is no job page.
- Currently there is no team page.


**What is the new behavior (if this is a feature change)?**
- A job page has been added.
- A team page has been added.


**Other information**:
- The version has been updated to `v1.1.0`.

Aichnerc committed:
Add job page

The job page has been implemented.

Aichnerc committed:
Add team page

The team page has been implemented.

Aichnerc committed:
Add SNEK button and links

@kleberbaum has been added. SNEK buttons have been added.

Aichnerc committed:
Improve testimonial card design

The background color of the cart and SNEK button have been changed.

Aichnerc committed:
Update version

v1.0.0 -> v1.1.0

sweggerat reviewed (approved):
good

Aichnerc committed:
Fix SNEK logo center

The SNEK logo in the SNEK button is now center.

pinterid commented on src/components/pages/TeamPage/index.jsx:
```diff
@@ -0,0 +1,176 @@
+      "https://avatars3.githubusercontent.com/u/21159914?s=460&u=afab4659183999f1adc85089bb713aefbf085b94&v=4",
+    facebook: "https://www.facebook.com/aichner.christian",
+    github: "https://github.com/Aichnerc",
+    snek: "https://snek.at/u/aichnerc",
```
```suggestion
    snek: "https://snek.at/u/Aichnerc",
```

pinterid reviewed (changes requested):
Found one issue.

Aichnerc committed:
Improve design

@pinterid has been added to the team and the design has been improved. LinkedIn has been added.

Theralun commented on src/components/pages/PositionsPage/index.jsx:
```diff
@@ -0,0 +1,219 @@
+                <div className="text-left">
+                  <p className="mb-2">Our main areas of research</p>
+                  <ul>
+                    <li>Avionik</li>
```
Avionics

Aichnerc replied:
Oh damn.

Theralun reviewed (changes requested):
Have a look.

Aichnerc committed:
Improve code quality

A typo has been fixed and some code quality changes have been introduced.

Theralun reviewed (approved):
LGTM.

kleberbaum reviewed (approved):
Should be ok to merge.
